### PR TITLE
e2e/cypress:fixed test cases

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
accessibility_nav_diff_regions_spec.js
keyboard_shortcuts_2_spec.js
message_display_mode_colorize_username_spec.js
accessibility_account_settings_spec.js

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/tests/integration/settings/display/message_display_mode_colorize_username_spec.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ describe('Settings > Display > Message Display: Colorize username', () => {
 function goToMessageDisplaySetting() {
     // # Go to Settings modal - Display section - Message Display
     cy.uiOpenSettingsModal('Display').within(() => {
-        cy.get('#displayButton').click();
+        cy.get('#displayButton').click().scrollIntoView();
```
Comment (yasserfaraazkhan) [inline e2e/cypress/tests/integration/settings/display/message_display_mode_colorize_username_spec.js:143]: @furqanmlk should it be 
cy.get('#displayButton').scrollIntoView().click();

**Review by yasserfaraazkhan — APPROVED** [review]
just a minor query about scrolling element into view and then performing click

**Review by Phrynobatrachus — APPROVED** [review]
Thank you!

**Pull request merged**